components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts

```diff
@@ -282,6 +282,7 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
   }
 
   ngOnDestroy() {
+    this.layoutFacade.ShowPageLoading(false);
```

Comment:
While this is a solution to the problem, I'm not sure if this is the optimal way of doing it.  I feel like killing the subscription should be accomplishing this on its own.